apps/sms/views/conversation/js/conversation.js

```diff
@@ -2009,7 +2077,8 @@ var ConversationView = {
     var bubble;
 
     do {
-      if (node.classList.contains('bubble')) {
+      // A Text node does not have a classList property
+      if (node.classList && node.classList.contains('bubble')) {
```

Comment:
Yes, I saw an error being thrown when long touching the late notice inside the bubble and I reproduced as well on the text message. It those cases the evt.target for some reason was the TextElement itself, so without classList prop.